### DISCUSSION TITLE
Unable to Upgrade from v1.3.2 to newer versions

### DISCUSSION THREAD

**Discussion by sfkkkkk** [Jyers/Marlin#1830] [Q&A]
Hi 
I have an Ender3 v2 with v4.2.2 board. 
In May last year (2021) I installed   "E3V2-ManualMesh-3x3-v4.2.2.bin" v1.3.2 without issue and have been using it since without issue (and I should say many thanks to Jyers as it is great). 

I today decided that I would upgrade to newest version, (so I downloaded .bin file, put it on 8MB SD Card, powered up). 
BUT I am finding that all I get is a black screen. It does not install (also checked this with octoprint that then says no serial connection so I presume not a simple screen issue)  

When I put the May2021 version, E3V2-ManualMesh-3x3-v4.2.2.bin" v1.3.2 back on card, it does install that and printer works again without issue.

Regarding versions I tried. IN chronological order I first tried installing   v2.0.1  and then  v1.3.3, v1.3.4, v1.3.5b   
Noting that I reinstalling v1.3.2 without issue between each of these that failed to install. 

Any suggestions to what I am doing wrong or missing?

Many thanks, SFKKKKKK

**Comment by BlueWings172** [comment]
@sfkkkkk 

Have you upgraded the screen firmware ?

If not, follow the steps in [this post](https://github.com/Jyers/Marlin/discussions/1768#discussioncomment-1943636) and try. You can choose one of the 3 available themes in Marlin/Display firmware/Firmware Sets/ and don't forget to rename the folder to DWIN after copying to SD card.

Here is a [video on how to open the screen housing](https://www.youtube.com/watch?v=Jswzrh2_ekk&ab_channel=Creality) :

**Comment by sfkkkkk** [comment]
With thanks to both.

Used two SD cards. Both formatted. Both work for  v1.3.2 but not others.

"Have you upgraded the screen firmware ?" cannot remember if i did it in MAy2021. But not since then, so will check and try that tonight. 
So that sounds a good likelyhood.
With thanks to both SFKKKKK

**Reply by K2Kevin** [reply]
Ive also seen weird inconsistencies with Firmware installs from SDcard.  I have always resolved by simply changing the file name to something like Firmware.bin.  Actually worked 100% of the time for me.  Strange...

**Reply by sfkkkkk** [reply]
Good point - but also tried that as i also believe Ender3 does not install new file if you use the same .bin file name as previous install.

**Reply by sfkkkkk** [reply]
I expect it is me not updatng the screen in last 12 months.

**Reply by K2Kevin** [reply]
Ive never updated screen firmware.

**Reply by BlueWings172** [reply]
Make sure the SD card is formatted in FAT32 - **4096** as mentioned above. This is critical for the screen update to work.

**Comment by marcribe** [comment]
Just make sure your new firmware has a different name to the firmware installed.
Re-flash by using a different filename (it needs to be always unique).
